sql/schema/v1-core.sql

```diff
@@ -620,77 +607,133 @@ REFERENCING NEW TABLE AS new_table
 FOR EACH STATEMENT
 EXECUTE FUNCTION after_v1_concurrency_slot_insert_function();
 
-CREATE OR REPLACE FUNCTION after_v1_concurrency_slot_delete_function()
-RETURNS trigger AS $$
+CREATE OR REPLACE FUNCTION cleanup_workflow_concurrency_slots(
```

Comment:
A future optimization could be to allow this function to take a set of records instead of just 1 -- but given the issues we've been seeing with `IN` queries, and given there's not a clean way to write this without that type of query, I figured it's slightly safer to just use a singleton.